test/functional/Java11andUp/src/org/openj9/test/condy/DefineCondyClass.java

```diff
@@ -0,0 +1,133 @@
+import static org.objectweb.asm.Opcodes.ARETURN;
+import static org.objectweb.asm.Opcodes.IRETURN;
+import static org.objectweb.asm.Opcodes.H_INVOKESTATIC;
+import static org.objectweb.asm.Opcodes.INVOKESPECIAL;
+import static org.objectweb.asm.Opcodes.RETURN;
+
+import org.objectweb.asm.ClassWriter;
+import org.objectweb.asm.*;
+
+import java.util.*;
+import java.io.*;
+
+/**
+ *
+ * Helper Class to create condyClass
+ *
+ */
+public class DefineCondyClass {
+	public static byte[] getNullCondyClassBytes(String name) {
+		ClassWriter cw = new ClassWriter(0);
+		cw.visit( 55, ACC_PUBLIC | ACC_SUPER, name, null, "java/lang/Object", null );
+		
+		MethodVisitor mv;
+		{
+			mv = cw.visitMethod( ACC_PUBLIC | ACC_STATIC, "getCondy", "()Ljava/lang/Object;", null, null );
+			mv.visitCode();
+			mv.visitLdcInsn( 
+				new ConstantDynamic( "constant_NULL", "Ljava/lang/Object;",
+					new Handle(
+							H_INVOKESTATIC, 
+							"java/lang/invoke/ConstantBootstraps", 
+							"nullConstant",
+							Type.getType( "(Ljava/lang/invoke/MethodHandles$Lookup;Ljava/lang/String;Ljava/lang/Class;)Ljava/lang/Object;").getDescriptor() 
+							)
+					)
+				);
+			mv.visitInsn( ARETURN );
+			mv.visitMaxs(1, 1);
+			mv.visitEnd();
+		}
+		cw.visitEnd();
+
+		return cw.toByteArray();
+	}
+
+	public static byte[] getPrimativeCondyClassBytes(String name) {
+		ClassWriter cw = new ClassWriter(0);
+		cw.visit( 55, ACC_PUBLIC | ACC_SUPER, name, null, "java/lang/Object", null );
+		
+		MethodVisitor mv;
+		{
+			mv = cw.visitMethod( ACC_PUBLIC | ACC_STATIC, "getCondy", "()I", null, null );
+			mv.visitCode();
+			mv.visitLdcInsn( 
+				new ConstantDynamic( "constant_int", "I",
+					new Handle(
+							H_INVOKESTATIC, 
+							"org/openj9/test/condy/BootstrapMethods", 
+							"bootstrap_constant_int",
+							Type.getType(
+								"(Ljava/lang/invoke/MethodHandles$Lookup;Ljava/lang/String;Ljava/lang/Class;I)I"
+								).getDescriptor()
+							),
+							123432
+					)
+				);
+			mv.visitInsn( IRETURN );
+			mv.visitMaxs(1, 1);
+			mv.visitEnd();
+		}
+		cw.visitEnd();
+
+		return cw.toByteArray();
+	}
+
+	public static byte[] getExceptionCondyClassBytes(String name) {
+		ClassWriter cw = new ClassWriter(0);
+		cw.visit( 55, ACC_PUBLIC | ACC_SUPER, name, null, "java/lang/Object", null );
+		
+		MethodVisitor mv;
+		{
+			mv = cw.visitMethod( ACC_PUBLIC | ACC_STATIC, "getCondy", "()I", null, null );
+			mv.visitCode();
+			mv.visitLdcInsn( 
+				new ConstantDynamic( "constant_int", "I",
+					new Handle(
+							H_INVOKESTATIC, 
+							"org/openj9/test/condy/BootstrapMethods", 
+							"bootstrap_constant_int_exception",
+							Type.getType(
+								"(Ljava/lang/invoke/MethodHandles$Lookup;Ljava/lang/String;Ljava/lang/Class;I)I"
+								).getDescriptor()
+							),
+							123432
+					)
+				);
+			mv.visitInsn( IRETURN );
+			mv.visitMaxs(1, 1);
+			mv.visitEnd();
+		}
+		cw.visitEnd();
+
+		return cw.toByteArray();
+	}
+}
```

Comment:
add newline.
I dont think it matters much for .java, but no harm in being consistent